src/firefox/index.js

```diff
@@ -264,11 +274,12 @@ export type CopyProfileOptions = {
  */
 export async function copyProfile(
   profileDirectory: string,
+  customPrefs?: FirefoxPreferences,
   {
+    app,
```

Comment:
Sure! I sorted them in function definition as well for consistency, will apply the same order in tests.